pkg/asset/cluster/aws/aws.go

```diff
@@ -12,13 +12,8 @@ import (
 func Metadata(clusterID string, config *types.InstallConfig) *aws.Metadata {
 	return &aws.Metadata{
 		Region: config.Platform.AWS.Region,
-		Identifier: []map[string]string{
-			{
-				"openshiftClusterID": clusterID,
-			},
-			{
-				fmt.Sprintf("kubernetes.io/cluster/%s", config.ObjectMeta.Name): "owned",
-			},
-		},
+		Identifier: []map[string]string{{
+			fmt.Sprintf("kubernetes.io/cluster/%s", clusterID): "owned",
+		}},
```

Comment:
This part right here looks like it will break deprovision for any pre-existing clusters. For better or worse, there are some. Is there any way we can work around that?

Reply:
CC @jhernand